src/neuroglancer/sliceview/decode_jpeg_stack.ts

```diff
@@ -20,14 +20,41 @@ import {Vec3, vec3} from 'neuroglancer/util/geom';
 export function decodeJpegStack(data: Uint8Array, chunkDataSize: Vec3) {
```

Comment:
Actually, it looks like it is called directly only once, in neuroglancer/sliceview/backend_chunk_decoders/jpeg.ts --- and from there you can get the number of channels as chunk.source!.spec.numChannels